Makefile

```diff
@@ -250,17 +250,14 @@ uninstall-prometheus:
 
 .PHONY: deploy-grafana
 deploy-grafana:
-	@echo "\n$(YELLOW)Local cluster $(CYAN)$(CLUSTER_GSLB1)$(NC)"
+	@echo "\n$(YELLOW)Local cluster $(CYAN)$(CLUSTER_NAME)1$(NC)"
 	@echo "\n$(YELLOW)install grafana $(NC)"
-	@$(eval PREVIOUS_CONTEXT := $(shell kubectl config current-context))
 	helm repo add grafana https://grafana.github.io/helm-charts
 	helm repo update
 	helm -n k8gb upgrade -i grafana grafana/grafana -f deploy/grafana/values.yaml \
-		--wait --timeout=2m0s \
-		--kube-context=k3d-$(CLUSTER_GSLB1)
-	kubectl config use-context k3d-$(CLUSTER_GSLB1)
-	kubectl apply -f deploy/grafana/dashboard-cm.yaml -n k8gb
```

Comment:
Don't we need dashboard-cm resource?
We're removing it later in `uninstall-grafana`, so it will fail

Reply:
it's there https://github.com/k8gb-io/k8gb/pull/797/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R260

Reply:
hmm, you're right, was confused by another error when running `make uninstall-grafana`:
```
uninstall grafana 
kubectl --context k3d-test-gslb1 delete -f deploy/grafana/dashboard-cm.yaml
Error from server (NotFound): error when deleting "deploy/grafana/dashboard-cm.yaml": configmaps "podinfo-dashboard" not found
make: [uninstall-grafana] Error 1 (ignored)
helm uninstall grafana -n k8gb --kube-context=k3d-test-gslb1
release "grafana" uninstalled
```